public/app/features/live/centrifuge/service.ts

```diff
@@ -168,16 +176,20 @@ export class CentrifugeService implements CentrifugeSrv {
    */
   getDataStream(options: LiveDataStreamOptions, config: LiveChannelConfig): Observable<DataQueryResponse> {
     return new Observable<DataQueryResponse>((subscriber) => {
+      const channelId = toLiveChannelId(options.addr);
```

Comment:
worth checking --- should the key be more than the channel?  the options include filter fields and buffer sizing instructions.

Curious what the reuseability is for multiple graphs pointing to the same channel, but with different fields filtered :thinking:

Reply:
> worth checking --- should the key be more than the channel? the options include filter fields and buffer sizing instructions.

yeah exactly - this was just initial implementation to test the approach


> Curious what the reuseability is for multiple graphs pointing to the same channel, but with different fields filtered 🤔

Atm I am testing the approach of having a single buffer on web worker side which takes in the superset of filters/buffer sizing options coming in from subscribers - I think it will work fine